lib/services/cloud-service.ts

```diff
@@ -161,4 +160,16 @@ export abstract class CloudService extends EventEmitter implements ICloudService
 			this.$logger.error(`Cloud operation ${cloudOperationId} failed with error: ${err.message}`);
 		}
 	}
+
+	private printMessageOnTheSameLine(msg: string): void {
+		const logger: any = this.$logger;
+		// Print the output on the same line to have cool effects like loading indicators.
+		// The cloud process will take care of the new lines.
+		if (logger.printInfoMessageOnSameLine) {
+			// Used in CLI before 6.0.0
+			logger.printInfoMessageOnSameLine(msg);
+		} else {
+			this.$logger.info(msg, {[this.$constants.LoggerConfigData.skipNewLine]: true });
```

Comment:
Is this going to print the message on the same line like `printInfoMessageOnSameLine `?

Reply:
yes